brian_cheung/week_two_challenges/more_methods.md

```diff
@@ -52,7 +82,9 @@ Example output:
 "The word 'dog' has 3 letters!"
 ```
 ```ruby
-# write your code here
+def informed_counter (word)
+  puts word.split('').count
+end
 ```
```

Comment:
Don't forget to read the directions fully: the output should return the string, "The word <word> has <number> letters!"